src/deprecated.js

```diff
@@ -0,0 +1,49 @@
+import { Counter, EnumOf, Integers, LowerCased, Strings } from './enum-xyz.js'
+
+/**
+ * @template T
+ * @param props {object}
+ * @param props.from {string}
+ * @param props.to {string}
+ * @param props.using {import('./enum-xyz.js').Enum<T>}
+ * @returns {import('./enum-xyz.js').Enum<T>}
+ */
+const deprecated = ({ from, to, using }) =>
+  EnumOf(
```

Comment:
This could also be a cool example in the example use cases folder where the ActionCreators is going. I did something similar to this with in another project recently, but I like this a lot.